sage-ios/SAGE/SAGE/BaseOperation.swift

```diff
@@ -15,4 +15,20 @@ class BaseOperation {
         let baseURL = NSURL(string: StringConstants.kBaseURL)
         return AFHTTPRequestOperationManager.init(baseURL: baseURL)
     }
+    
+    static func loadSchools(completion: ((NSMutableArray) -> Void)){
+        
+        let operationManager = AFHTTPRequestOperationManager()
```

Comment:
user the manager() method above. feel free to add the serializer stuff if you want to, i dont think its needed?